src/app/clusters/ota-requestor/DefaultOTARequestor.h

```diff
@@ -198,8 +198,7 @@ class DefaultOTARequestor : public OTARequestorInterface, public BDXDownloader::
     /**
      * Record the error update state by informing the driver of the error and calling `RecordNewUpdateState`
```

Comment:
We're technically no longer informing the driver directly here.
```suggestion
     * Record the error update state and transition to the idle state
```

Reply:
Made the change